drivers/SmartThings/matter-thermostat/fingerprints.yml

```diff
@@ -8,7 +8,7 @@ matterManufacturer:
   - id: "Leedarson AiDot Air Purifier"
     deviceLabel: AiDot Air Purifier
     vendorId: 0x1396
-    productId: 0x0043A
+    productId: 0x043A
```

Comment:
Just as a heads up, this is only a formatting change, functionally this is the same value. Just wanted to confirm that the value was not supposed to change as well

Reply:
Yes - that is the same as my intention and understanding. Thank you.